store_db.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE book_store (
+      id     INTEGER PRIMARY KEY,
```

Comment:
The value is that you're not creating the values. Postgres is doing it for you. In a production environment, you will not be adding the id. The database will be doing that for the user.